readme.md

```diff
@@ -311,6 +311,25 @@ You can also use your own local Babel version:
 }
 ```
 
+#### Transpiling Dependent Modules
+
+*Note:  AVA currently only transpiles the tests that you ask it to run.  In other words, it won't transitively transpile modules that you ```import``` from outside of the test (that also may happen to be written in ES2015 and/or use the ES2015 module syntax).  While there are valid reasons for taking this approach (no impact/modification to the code under test), it may not be what you expect!*
```

Comment:
A few nits:

- IMO, you are overusing *emphasis* here. Emphasizing whole paragraphs is overkill. Emphasize the one/two sentences I need to read to understand what the paragraph is about, so I can move on if it does not concern me.

- ... run. In other words, it won't   `=>` ... run. It will **not** (`In other words` is superfluous language)

- `transitively transpile` - let's think of a simpler way to say that 

> Note: *AVA currently only transpiles the tests that you ask it to run.* It will **not** ...